packages/core/src/utils/FetchUtil.ts

```diff
@@ -94,6 +97,9 @@ export class FetchUtil {
         }
       })
     }
+    if (this.clientId) {
```

Comment:
@tomiir Is the `clientId` required param? Should we return any error if it's not set correctly?

Reply:
No, it's only set on WC relay connections so it might not be set